main.tf

```diff
@@ -48,7 +39,7 @@ resource "aws_db_instance" "db" {
   storage_type              = "${var.storage_type}"
   storage_encrypted         = "${var.storage_encrypted}"
   name                      = "${var.name}"
-  final_snapshot_identifier = "${var.environment}-${var.name}-final-${random_id.timestamp.hex}"
```

Comment:
I think we should give the user of the module some mechanism of flexibility. So I opt for adding en optional suffix that can be provided by the user of the module. This suffix will be added to the final_snapshot_identifier. Name of variable should be `final_snapshot_identifier_suffix`. Change requires as well an update of the README.